pages/book/statements.mdx

```diff
@@ -8,14 +8,18 @@ The following statements can appear anywhere in the [function](/book/functions)
 
 The `let{:tact}` statement allows local and [block](#block)-scoped variable declaration.
 
-In Tact, declaring a variable always requires an initial value and an explicit type:
+In Tact, declaring a local variable always requires an initial value. However, the type ascription can be omitted and Tact will try to infer it from the initial value:
 
 ```tact
-let value: Int = 123;           // declaration
-let valueOptional: Int? = null; // Int or null
+let value: Int = 123; // full declaration with type and value
+let vInferred = 123;  // inferred type Int
 
-let valueMap: map<Int, Int> = emptyMap();
-let valueMapWithSerialization: map<Int as uint8, Int as uint8> = emptyMap();
+let vOptional: Int? = null;           // explicit type Int or null
+let vMap: map<Int, Int> = emptyMap(); // explicit type map<Int, Int>
```

Comment:
it should be noted that without explicit type annotation this won't typecheck because we don't know the concrete key and value types at this point